src/obs-websocket.h

```diff
@@ -19,6 +19,115 @@ with this program. If not, see <https://www.gnu.org/licenses/>
 
 #pragma once
 
+/**
+ * @typedef Filter
+ * @property {Boolean} filterEnabled
+ * @property {Number} filterIndex
+ * @property {String} filterKind
+ * @property {String} filterName
+ * @property {Object} filterSettings
+ * @api typedefs
+ */
+
+/**
+ * @typedef MeterData
+ * @property {String} inputName
+ * @property {Array<Array<Number>>} inputLevelsMul
+ * @api typedefs
+ */
+
+/**
+ * @typedef SceneItemTransform
+ * @property {Number} sourceWidth
+ * @property {Number} sourceHeight
+ * @property {Number} positionX
+ * @property {Number} positionY
+ * @property {Number} rotation
+ * @property {Number} scaleX
+ * @property {Number} scaleY
+ * @property {Number} width
+ * @property {Number} height
+ * @property {Number} alignment
+ * @property {Number} boundsType
+ * @property {Number} boundsAlignment
+ * @property {Number} boundsWith
+ * @property {Number} boundsHeight
+ * @property {Number} cropLeft
+ * @property {Number} cropRight
+ * @property {Number} cropTop
+ * @property {Number} cropBottom
+ * @api typedefs
+ */
+
+/**
+ * @typedef SceneItem
+ * @property {Number} sceneItemId
+ * @property {Number} sceneItemIndex
+ * @property {Boolean} sceneItemEnabled
+ * @property {Boolean} sceneItemLocked
+ * @property {SceneItemTransform} sceneItemTransform
+ * @property {Number} sceneItemBlendMode
+ * @property {String} sourceName
+ * @property {Number} sourceType
+ * @property {String|undefined} inputKind
+ * @property {Boolean|undefined} isGroup
+ * @api typedefs
+ */
+
+/**
+ * @typedef Scene
+ * @property {String} sceneName
+ * @property {Number} sceneIndex
+ * @api typedefs
+ */
+
+/**
```

Comment:
I would personally think it would be good to include other fields not explicitly returned in a request as an object in these typedefs. For example, `inputSettings` is never included as an `Input` object, but is very much associated with inputs. Let me know your thoughts.

Reply:
I can go through and type everything currently typed as an `Object`, if you like. Sound good?